activerecord/lib/active_record/fixtures.rb

```diff
@@ -585,6 +585,14 @@ def identify(label, column_type = :integer)
         end
       end
 
+      def signed_global_id(fixture_set_name, label, column_type: :integer, **options)
```

Comment:
@georgeclaghorn do you feel like additions to _both_ the ActiveRecord side and the introduction of the ActionText side are worthwhile?

Would this be better as an ActionText only change? An ActiveRecord only change that enables people to declare their own `<action-text-attachment>` elements manually?